backend/src/web-server/controllers/oibus.controller.ts

```diff
@@ -59,8 +64,13 @@ export default class OibusController extends AbstractController {
     const { name } = ctx.request.query;
     const file = ctx.request.file;
     if (name && file) {
+      const externalSource = ctx.app.repositoryService.externalSourceRepository.findExternalSourceByReference(name as string);
+      if (!externalSource) {
+        return ctx.notFound();
+      }
+
```

Comment:
The default behaviour is to store the data, even if the external source is missing. This way, we can still have data sent to connectors without subscription, but keeping trace of the external source in the log. 
I think we should keep it like this behaviour if possible ?